src/i18n/l10n/en.ftl

```diff
@@ -879,5 +924,19 @@ Failed-to-log-in = Failed to log in
 # Generic error message
 Something-went-wrong = Something went wrong.
 
+#Geoprivacy sheet descriptions
+Anyone-using-iNaturalist-can-see = Anyone using iNaturalist can see where this species was observed, and scientists can most easily use it for research.
+The-exact-location-will-be-hidden = The exact location will be hidden publicly, and instead generalized to a larger area. (Threatened and endangered species are automatically obscured).
+The-location-will-not-be-visible = The location will not be visible to others, which means it may be difficult to identify.
+
+#Wild status sheet descriptions
```

Comment:
FYI, comments in FTL refer only to the following line and are meant to provide context to translators just for the following string, so try to make sure comments a) refer to the following string only, and b) help translators translate that string.

As tempting as it is to try and group translations logically, there is no grouping that is logical to everyone, so I'd just add new translations to the end of the file for now. Ultimately I think we should alphabetize them, but I think it makes more sense to automate that in a pre-commit hook (will require parsing the FTL so we can keep the comments in the right places).

Reply:
There is a mention of `##` for block comments in the fluent docs.